### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by mohawk2**
This simplifies the one test, adds metadata pointing to this updated GH repo, and stops the module from breaking in a "place with a space".

### CODE REVIEW

**Comment by karlglazebrook** [comment]
I tested this, think we should merge it.

File: F77.pm [hunk 1]
```diff
@@ -108,7 +109,7 @@ $F77config{MinGW}{G77}{Link} = sub {
       $dir = "/usr/local/lib";
       $lib = "f2c";
    }  
-   return( "-L$dir -L/usr/lib -l$lib -lm" );
+   return( "\"-L$dir\" -L/usr/lib -l$lib -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
   return( qq{"-L$dir" -L/usr/lib -l$lib -lm} );
```

File: F77.pm [hunk 2]
```diff
@@ -123,7 +124,7 @@ $F77config{MinGW}{GFortran}{Link} = sub {
    } else {
       $dir = "/usr/local/lib";
    }    
-   return( "-L$dir -L/usr/lib -lgfortran -lquadmath -lm" );
+   return( "\"-L$dir\" -L/usr/lib -lgfortran -lquadmath -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
   return( qq{"-L$dir" -L/usr/lib -lgfortran -lquadmath -lm} );
```

File: F77.pm [hunk 3]
```diff
@@ -142,7 +143,7 @@ $F77config{Sunos}{F77}{Link} = sub {
    $dir = find_highest_SC("/usr/lang/SC*");
    return "" unless $dir; # Failure
    print "$Pkg: Found Fortran latest version lib dir $dir\n";
-   return "-L$dir -lF77 -lm";
+   return "\"-L$dir\" -lF77 -lm";
```
Comment (djerius) [inline F77.pm]:
```suggestion
   return qq{"-L$dir" -lF77 -lm};
```

File: F77.pm [hunk 4]
```diff
@@ -361,7 +362,7 @@ $F77config{Freebsd}{G77}{Link} = sub {
     } else {
         $dir = "/usr/local/lib";
     }    
-    return( "-L$dir -L/usr/lib -lg2c -lm" );
+    return( "\"-L$dir\" -L/usr/lib -lg2c -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
    return( qq{"-L$dir" -L/usr/lib -lg2c -lm} );
```

File: F77.pm [hunk 5]
```diff
@@ -376,7 +377,7 @@ $F77config{Freebsd}{GFortran}{Link} = sub {
     } else {
         $dir = "/usr/local/lib";
     }    
-    return( "-L$dir -L/usr/lib -lgfortran -lm" );
+    return( "\"-L$dir\" -L/usr/lib -lgfortran -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
    return( qq{"-L$dir" -L/usr/lib -lgfortran -lm} );
```

File: F77.pm [hunk 6]
```diff
@@ -702,7 +703,7 @@ sub gcclibs {
       unless($fallback_compiler eq 'GFortran' && $^O =~ /MSWin/i) {
          $gccdir = `$gcc -m32 -print-libgcc-file-name`; chomp $gccdir;
          $gccdir =~ s/\/libgcc.a//;
-         return " -L$gccdir -lgcc";
+         return " \"-L$gccdir\" -lgcc";
```
Comment (djerius) [inline F77.pm]:
```suggestion
         return qq{"-L$gccdir" -lgcc};
```

File: F77.pm [hunk 7]
```diff
@@ -789,9 +790,9 @@ sub link_gnufortran_compiler {
    if ( $Config{osname} =~ /darwin/ && $Config{osvers} >= 14
       && $compiler eq 'gfortran' && $version >= 4.9 ) { # Add extra libs for gfortran versions >= 4.9 and OS X
       $append = "-lgcc_ext.10.5 -lgcc_s.10.5 -lquadmath";
-      return( "-L$dir $append -L/usr/lib -l$lib -lm" );
+      return( "\"-L$dir\" $append -L/usr/lib -l$lib -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
      return( qq{"-L$dir" $append -L/usr/lib -l$lib -lm} );
```

File: F77.pm [hunk 8]
```diff
@@ -789,9 +790,9 @@ sub link_gnufortran_compiler {
...
    }
-   return( "-L$dir -L/usr/lib -l$lib -lm" );
+   return( "\"-L$dir\" -L/usr/lib -l$lib -lm" );
```
Comment (djerius) [inline F77.pm]:
```suggestion
   return( qq{"-L$dir" -L/usr/lib -l$lib -lm} );
```

**Review by djerius — COMMENTED** [review]
I suggest replacing` "\"" `with `qq{"}` for clarity.

File: F77.pm (hunk 8 above)
Comment (mohawk2) [reply F77.pm]: Agreed, all implemented!

**Comment by mohawk2** [comment]
@karlglazebrook Have you heard from @devel-chm lately? It's been a while.

**Comment by karlglazebrook** [comment]
No I have not heard from him either. Hope all is OK.

**Comment by devel-chm** [comment]
Hi All-

I've been very busy with no time for PDL work.

--Chris

**Comment by karlglazebrook** [comment]
No worries. Me too for the last few months.

If you have 5 secs to up my github privs on pdl/extutils-f77 that would be welcome.

Karl

**Comment by mohawk2** [comment]
@karlglazebrook I think you may already have rights to merge the PRs on anything in PDLPorters, including this. It looks like you also have the rights to release this (and other PDL-ish modules) onto CPAN. Do you feel like doing so?

**Comment by karlglazebrook** [comment]
It seems not, at least for PDLPorters/extutils-f77, see screenshot

can someone fix?

Karl

**Comment by mohawk2** [comment]
@karlglazebrook Unfortunately the email->issue gateway drops attachments. Could you come to the web page and attach the screenshot? I've had a quick look and can't see anything that shows why you wouldn't be able to merge; you're showing here as a "Member" of PDLPorters.

**Comment by mohawk2** [comment]
Breaking news: Karl was only a "member" of the organisation, but is now an "owner". You should be able to merge now? Are you also able to cut releases?

**Comment by karlglazebrook** [comment]
Merging this. All seems like good changes.

**Comment by karlglazebrook** [comment]
Yes! Thanks for doing that

I’ve merged your tweaks branch

Karl

**Comment by mohawk2** [comment]
@karlglazebrook Great! Are you minded to release EU::F77 to CPAN? :-)

**Comment by karlglazebrook** [comment]
Not yet. I have someone threatening me with new patches which sound useful :)

Is there a cool way to go github->CPAN?

**Comment by mohawk2** [comment]
You could release this one, and *then* take your time with the other patches. There's no cost per release :-)

To answer your question, there are various tools these days for easy uploading. However, I tend to stay old-school, and after running the various normal tests, I then do these steps manually:
- increment the version
- update `Changes` as appropriate
- re-run `perl Makefile.PL`
- `git commit -am v1.21`
- `git tag v1.21`
- `make disttest`
- `make dist`
- `cpan-upload ExtUtils-*`
- `rm ExtUtils-*`
- `git push`
- `git push --tags`

**Comment by karlglazebrook** [comment]
It sounds like it would only be a day or two and a very minor patch


Yes I can do all that, I was hoping there would be a button on CPAN or GITHUB to release to CPAN :)

**Comment by mohawk2** [comment]
I didn't doubt you! I was just noting it down for myself. I should really automate all of that.

Sadly, no cool button for now. It would probably be possible to make Travis take every successful build of `master` and release it as a new version on CPAN, but not quite trivially.

**Comment by karlglazebrook** [comment]
Travis?

**Comment by mohawk2** [comment]
Sorry - continuous integration service. It's not even been set up for this repo. One possibility with continuous integration is to do follow-on steps on success. With enough tests, one can reach a level of software infrastructure known as "continuous delivery" where a merge leads to a newly-released version. We aren't near there yet with this repo! Quite a few things would need adjusting.

**Comment by karlglazebrook** [comment]
Let’s not go there yet!

Karl

**Pull request merged**